docs/extend/k8s-extend.md

```diff
@@ -0,0 +1,36 @@
+## Import New Component
```

Comment:
## Importing New Component [Prajakta: There is inconsistency in the way titles are written in Zowe documentation. I suggest use of gerund form. Also, some titles are in title case and some in sentence case. Here, since we have used title case, I have also suggested title case.